src/vivarium_nih_us_cvd/constants/data_values.py

```diff
@@ -63,6 +64,7 @@ class __TimestepCleanupPriorities(NamedTuple):
 
     HEALTHCARE_VISITS: int = 5
     TREATMENT: int = 6
+    OUTREACH: int = 7
```

Comment:
This is never used.

Reply:
Ah, lingering from when I tried haveing OutreachEffect initialize/manage the 'outreach' column.